src/requests/typings/deliveryFunction.ts

```diff
@@ -27,7 +28,9 @@ export type GraphQLFieldTypeMapping = {
 }
 
 export type GraphQLQueryRequest = {
+  type: DeliveryFunctionRequestEventType.GRAPHQL_QUERY
   query: string
+  isIntrospectionQuery: boolean
```

Comment:
Should this also be part of Field mapping too?

Reply:
I'm not sure if I'm following. This boolean is used to indicate wether the `query` being sent for this event is an introspection query. An app can use this to easily overwrite where the schema is loaded from, for instance by loading it from the file system instead of passing the introspection query along to the 3rd party GraphQL API.